README.md

```diff
@@ -55,7 +55,7 @@ segments:
 Kinds may or may not apply/be adopted for all platforms. For version 1 of the segmentation of HALO flights, one or more of the following kinds are assigned to each segment:
 
 ### circle:
-- can be identified by a constant change of aircraft heading, a duration of about 60 min , as well as a roughly constant roll angle of 1.5-3 degrees (positive or negative, depending on turning clockwise or counter-clockwise) for the standard circle of about 200km diameter. Duration and roll angle change with the chosen circle radius. The smaller ATR circle has about 140km diameter and a larger roll angle of 3-5 degree depending on and varying with the wind speed and direction at the respective altitude.
+- can be identified by a constant change of aircraft heading, a duration of about 60 min , as well as a roughly constant roll angle of 1.5-3 degrees (positive or negative, depending on turning clockwise or counter-clockwise) for the standard circle with a radius of about 133km. Duration and roll angle change with the chosen circle radius. The smaller ATR circle has a radius of about 59km and a larger roll angle of 3-5 degree depending on and varying with the wind speed and direction at the respective altitude.
```

Comment:
The smaller sized circles typically for ATR / METEOR coordination had a radius of about 72km if I remember it correctly (definitely so on 2024-09-23). I assume that you took that value from a flight that you were segmenting which makes me think that we might want to include the fitted circle also in the plot of any circle segments to see whether the derived radius matches and if not, implement a possibility to adjust manually until it fits by eye?

Reply:
PR #79 can help check the approximated circle radii

Reply:
After another segmented flight, I've seen various values between 59 and 72km. Thus, I would suggest, we just provide the approximate value of 70km. If you agree, please feel free to merge :)